src/applications/representatives/components/PoaRequestsTable/PoaRequestsTable.jsx

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { acceptDeclinePOARequest } from '../../actions/poaRequests';
+
+const PoaRequestsTable = ({ poaRequests }) => {
+  return (
+    <va-table data-testid="poa-requests-table" sort-column={1}>
+      <va-table-row slot="headers">
+        <span>Claimant</span>
+        <span>Submitted</span>
+        <span>Description</span>
+        <span>Status</span>
+        <span>Actions</span>
+      </va-table-row>
+      {poaRequests.map(({ id, name, date, description, status }) => (
+        <va-table-row key={id}>
+          <span data-testid={`${id}-claimant`}>{name}</span>
+          <span data-testid={`${id}-submitted`}>{date}</span>
+          <span data-testid={`${id}-description`}>{description}</span>
+          <span data-testid={`${id}-status`}>{status}</span>
+          <span>
+            {status === 'Pending' && (
+              <>
+                <va-button
+                  data-testid={`${id}-accept-button`}
+                  secondary
+                  text="Accept"
+                  onClick={() => acceptDeclinePOARequest(id, 'accept')}
+                />
+                <va-button
+                  data-testid={`${id}-decline-button`}
+                  secondary
+                  text="Decline"
+                  onClick={() => acceptDeclinePOARequest(id, 'decline')}
+                />
```

Comment:
While we're here, can we find a way to make it so that the caller doesn't need to know about any special strings like `'accept'` and `'decline'`? One way to accomplish that would be to export 2 functions: `acceptPOARequest` and `declinePOARequest`. Even if we end up preferring some other form, I think it could worthwhile to express this basic design principle in the code early so that people are encouraged to continue following it later.

Reply:
Looks good!